shell/platform/darwin/ios/framework/Headers/FlutterPlatformViews.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERPLATFORMVIEWS_H_
+#define FLUTTER_FLUTTERPLATFORMVIEWS_H_
+
+#import <CoreMedia/CoreMedia.h>
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+#include "FlutterCodecs.h"
+#include "FlutterMacros.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ An instance of a `UIView` created by a `FlutterPlatformViewFactory`.
+
+ This protocol is used to associate a view with a dispose callback.
+ */
+FLUTTER_EXPORT
+@protocol FlutterPlatformView <NSObject>
+- (UIView*)view;
+/**
+ Called when the Flutter engine no longer needs this `view`.
+
+ The implementation of this method should release any kept for this view.
+ */
+- (void)dispose;
+@end
+
+FLUTTER_EXPORT
+@protocol FlutterPlatformViewFactory <NSObject>
+/**
+ Create a `FlutterPlatformView`.
+
+ Implemented by iOS code that expose a `UIView` for embedding in a Flutter app.
+
+ The implementation of this method should create a new `UIView` and return a `FlutterPlatformView`
+ that wraps it.
+
+ - Parameters:
```

Comment:
This format won't really work in our docs.  See https://github.com/flutter/engine/pull/6447/files#diff-2acb8a0db370213c3467450b9e70984dR43 for example

Reply:
(`@param` can be used multiple times, and will format as a table)

Reply:
done